models/gyroscope/arbitrum/gyroscope_arbitrum_schema.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+
+models:
+
+  - name: gyroscope_arbitrum_trades
+
+    meta:
+
+      blockchain: arbitrum
+      project: gyroscope
+      contributors: fmarrr
+
+    config:
+
+      tags: ["arbitrum", "gyroscope", "trades"]
+
+    description: trades on gyroscope E-CLP pools on arbitrum
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
```

Comment:
this differs from unique key in model, where it's `block_time`
either one should work, but please keep consistent between the two spots